### DISCUSSION TITLE
Does it support WKWebview in SwiftUI iOS 12+

### DISCUSSION THREAD

**Discussion by Jalil-Irfan** [siteline/SwiftUI-Introspect#249] [Q&A] [answered]
I have a WKWebView where i have the page scroll bounce , showing unwanted white space in the top and bottom edge , when ever the scrolling content reaches the end.
I want to avoid this scrolling will this Introspect full fill my requirement

**Comment by raheelahmad** [comment, marked as answer]
How are you using `WKWebView`? If you are embedding it via `UIViewRepresentable `, you should already have access to it.
